components/kubeflow/launcher/Dockerfile

```diff
@@ -11,12 +11,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM ubuntu:16.04
 
-RUN apt-get update -y && \
-    apt-get install --no-install-recommends -y -q ca-certificates python-dev python-setuptools wget && \
-    easy_install pip && \
-    pip install pyyaml==3.12 kubernetes
```

Comment:
What pyyaml version will we use after this change?

Reply:
It would be the latest stable release. (5.4.1 as of today)
pyyaml 3.12 only supports up to Python 3.5: https://pypi.org/project/PyYAML/3.12/, so we'll have to upgrade it as part of this change.
All we use from pyyaml is the `safe_load` function, I think any new version should be just fine.
https://github.com/kubeflow/pipelines/blob/32ce8d8f90bfc8f89a2a3c347ad906f99ba776a8/components/kubeflow/launcher/src/launch_tfjob.py#L30

Reply:
Thank you Chen for the usage of pyyaml! Then it is fine using latest stable version.